### DISCUSSION TITLE
Hugo - static site generator

### DISCUSSION THREAD

**Discussion by Bammede** [coollabsio/coolify#669] [Feature Requests]
Please add Hugo - static site generator

**Comment by lcfd** [comment]
I think that the best place for asking new services is

https://feedback.coolify.io/

**Reply by luckydonald** [reply]
That place is 404.

**Reply by lcfd** [reply]
Hi @luckydonald, the comment is from 2022.

@andrasbacsai may have moved the service to another place, or dismissed it completely.
I'm not up-to-date about that.

**Reply by KoljaL** [reply]
There is a feedback icon inside the coolify app. (left bottom)
